### DISCUSSION TITLE
Does DCM4CHEE-ARC-LIGHT support any form of high availability, such as load balancing, fail-over or cluster?

### DISCUSSION THREAD

**Discussion by leon1227** [dcm4che/dcm4chee-arc-light#4083] [Q&A] [answered]
Already done quite some searches without firm answers... Trying to find out if dcm4chee-arc-light could be deployed in any form of high availability, such as load balancing, fail-over or cluster. Tried with v5.30. It's easy to build a cluster of database. But how to set up dcm4chee-arc-light with fail-over or cluster?

**Comment by robert-khuh** [comment]
Any update on this question?

**Comment by vrindanayak** [comment, marked as answer]
Yes, all of the above. We support services for that in the professional version of the archive.
